docs/raw/serving-community-history.md

```diff
@@ -0,0 +1,406 @@
+
+## Message archive distribution
+
+Message archives are available via the BitTorrent network as they are being [seeded by the community owner node](#seeding-message-history-archives).
+Other community member nodes will download the message archives from the BitTorrent network once they receive a magnet link that contains a message archive index.
+
+The community owner node MUST send magnet links containing message archives and the message archive index to a special community channel. The topic of that special channel follows the following format:
+
+```
+/{application-name}/{version-of-the-application}/{content-topic-name}/{encoding}
+```
+
+All messages sent with this topic MUST be instances of `ApplicationMetadataMessage` ([6/PAYLOADS](/specs/6-payloads)) with a `payload` of `CommunityMessageArchiveIndex`.
+
+Only the community owner MAY post to the special channel. Other messages on this specified channel MUST be ignored by clients.
+Community members MUST NOT have permission to send messages to the special channel.
+However, community member nodes MUST subscribe to special channel to receive waku messages containing magnet links for message archives.
+
+## Canonical message histories
+
+Only community owners are allowed to distribute messages with magnet links via the special channel for magnet link exchange. Community members MUST NOT be allowed to post any messages to the special channel.
+
+Status nodes MUST ensure that any message that isn't signed by the community owner in the special channel is ignored.
+
+Since the magnet links are created from the community owner node's database (and previously distributed archives), the message history provided by the community owner becomes the canonical message history and single source of truth for the community.
+
+Community member nodes MUST replace messages in their local databases with the messages extracted from archives within the same time range. Messages that didn't receive the community owner node MUST be removed and are no longer part of the message history of interest, even if it already existed in a community member node's database.
+
+
+## Fetching message history archives
+
+Generally, fetching message history archives is a tree step process:
+
+1. Receive message archive index magnet link as described in [Message archive distribution], download `index` file from torrent, then determine which message archives to download
+3. Download individual archives
+
+Community member nodes subscribe to the special channel that community owner nodes publish magnet links for message history archives to. There are two scenarios in which member nodes can receive such a magnet link message from the special channel:
+
+1. The member node receives it via live messages, that is, messages that are relayed by store nodes 
```

Comment:
Store nodes are known for their storage service as part of which they also relay messages by running waku relay protocol. So, store nodes are a subset of relay nodes but not all of them. Waku relay nodes are the ones that run the relay protocol. 
I think it might be better to skip this explanation as it may cause confusion for the readers.

```suggestion
1. The member node receives it via live messages by listening to the special channel
```